assembl/fabfile.py

```diff
@@ -666,6 +666,13 @@ def get_db_dump_name():
     return 'assembl-backup.pgdump'
 
 
+def get_versioned_db_dump_name():
+    bumpversion_config = SafeConfigParser()
```

Comment:
I think the fabfile is already quite big, can we put the backup part in a different file?

Reply:
je suis pour un éclatement et un nettoyage globale du fabfile... à discuter